internal/pkg/deploy/cloudformation/app.go

```diff
@@ -99,11 +99,11 @@ func (cf CloudFormation) upgradeAppStack(s *cloudformation.Stack) error {
 			_ = cf.cfnClient.WaitForUpdate(context.Background(), s.Name)
 			continue
 		}
-		s.Parameters = descr.Parameters
+		// We only need the tags from the previously deployed stack.
```

Comment:
why?

Reply:
because for app upgrade we don't need to update the tags we can just use the previous since `app upgrade` won't be required to update any tags for now. However, for both template and parameters we want to update to the latest. Does it make sense?

Reply:
I see that makes sense, what's a sample Parameter that would change ?

Reply:
the new param we add to the stack `AppDomainHostedZoneID`